process/process_linux.go

```diff
@@ -455,10 +509,15 @@ func (p *Process) MemoryMaps(grouped bool) (*[]MemoryMapsStat, error) {
 
 	blocks := make([]string, 16)
 	for _, line := range lines {
-		field := strings.Split(line, " ")
+		if len(line) == 0 {
+			continue
+		}
+
+		field := strings.SplitN(line, " ", 6)
 		if strings.HasSuffix(field[0], ":") == false {
 			// new block section
 			if len(blocks) > 0 {
+				field[len(field)-1] = strings.TrimSpace(field[len(field)-1])
```

Comment:
trim only from the left side as the filename could include spaces from the right side